client/components/category-pill-navigation/style.scss

```diff
@@ -24,7 +24,13 @@
 		display: flex;
 		gap: 16px;
 		align-items: center;
+		// Chrome, Edge, Firefox
 		scrollbar-width: none;
+
+		// Safari
+		&::-webkit-scrollbar {
+			display: none;
+		}
```

Comment:
This isn't related to the core changes here, but it fixes a regression that #88710 introduced. Apparently, Safari doesn't support `scrollbar-width`

Reply:
Confirming that the scroll bar is no longer shown in Safari:

|Before|After|
|-------|-----|
|<img width="1221" alt="Screen Shot 2024-03-21 at 11 19 36 AM" src="https://github.com/Automattic/wp-calypso/assets/104910361/3678a1c9-1b5a-48ce-81dc-40dd9bace6c1">|<img width="1224" alt="Screen Shot 2024-03-21 at 11 19 47 AM" src="https://github.com/Automattic/wp-calypso/assets/104910361/29354a18-e479-49f1-8b90-5dc6f2681130">|